changelog.json

```
@@ -1,5 +1,28 @@
 {
   "releases": {
+    "2.9.4": [
+      "[Added] Add syntax mapping for HAML - #13009. Thanks @alexanderadam!",
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Fixed] Diffs are scrolled to the top when switching between files - #12980",
+      "[Fixed] Fix SSH prompt for unknown hosts in some scenarios - #13050",
+      "[Fixed] Apply syntax highlighting when viewing a small change for the first time - #13004",
+      "[Fixed] Wrap long email addresses in the misattributed commit warning popover - #13044",
+      "[Fixed] Refresh diffs when application receives focus - #12962",
+      "[Fixed] Tokens are not considered invalid when accessing GitHub Enterprise from outside of the company's VPN - #12943",
```

I'm not wedded to this, feel free to reject or reword if you don't like this.
```suggestion
      "[Fixed] Only consider tokens invalid when 401 error comes from GitHub Enterprise - #12943",
```